plugins/inputs/github/README.md

```diff
@@ -0,0 +1,45 @@
+# GitHub Input Plugin
+
+The [GitHub](https://www.github.com) input plugin gathers statistics from GitHub repositories.
+
+### Configuration:
+
+```toml
+[[inputs.github]]
+    repositories = [
+      "owner/repository",
+    ]
+
+    ## Optional: Unauthenticated requests are limited to 60 per hour
+    access_token = "personal access token"
+
+    ## Optional: Default 5s
+    http_timeout = "5s"
+```
+
+### Metrics:
+
+- github_repository
+  - tags:
+    - `full_name` - The full name of the repository, including owner/organization
+    - `name` - The repository name
+    - `owner` - The owner of the repository
+    - `language` - The primary language of the repository
+    - `license` - The license set for the repository
+  - fields:
+    - `stars` (int)
+    - `forks` (int)
+    - `open_issues` (int)
+    - `size` (int)
+- github_rate_limit
```

Comment:
This meta metric would best be reported as an `internal` (aka `selfstat`) metric.  These are only emitted if the `internal` plugin is enabled, check the statsd plugin for an example.